src/scrcmd_8.c

```diff
@@ -0,0 +1,88 @@
+#include "scrcmd.h"
+#include "fieldmap.h"
+#include "friend_group.h"
+#include "unk_0202C730.h"
+#include "unk_0203E348.h"
+#include "math_util.h"
+
+BOOL ScrCmd_683(SCRIPTCONTEXT *ctx) {
+    u32 *unkC = FieldSysGetAttrAddr(ctx->fsys, SCRIPTENV_0C);
+    u16 *var0 = ScriptGetVarPointer(ctx);
+    *var0 = *unkC;
+    return TRUE;
+}
+
+BOOL ScrCmd_465(SCRIPTCONTEXT *ctx) {
+    MSGFMT **msg = FieldSysGetAttrAddr(ctx->fsys, SCRIPTENV_MSGFMT);
+    SAV_FRIEND_GRP *group = Save_FriendGroup_get(ctx->fsys->savedata);
+    SAVEDATA *save = ctx->fsys->savedata;
+
+    u16 var = ScriptReadHalfword(ctx);
+    switch (var) {
+    case 0: {
+        u16 var0 = ScriptGetVar(ctx);
+        u16 *retPtr = ScriptGetVarPointer(ctx);
+        *retPtr = sub_0202C860(group, var0);
+        return FALSE;
+    }
+    case 1: {
+        u16 var0 = ScriptGetVar(ctx);
+        u16 *retPtr = ScriptGetVarPointer(ctx);
+        *retPtr = sub_0202C878(group, var0);
+        return FALSE;
+    }
+    case 2: {
+        u16 groupId = ScriptGetVar(ctx);
+        u16 fieldNo = ScriptGetVar(ctx);
+        BufferGroupName(*msg, save, groupId, fieldNo, 0);
+        break;
+    }
+    case 3: {
+        u16 groupId = ScriptGetVar(ctx);
+        u16 fieldNo = ScriptGetVar(ctx);
+        BufferGroupName(*msg, save, groupId, fieldNo, 1);
+        break;
+    }
+    case 4: {
+        u16 *r5 = sub_0202C7E0(group, 0, 0);
+        u16 *retPtr = ScriptGetVarPointer(ctx);
+        CreateNamingScreen(ctx->taskman, 5, 0, 7, 0, r5, retPtr);
```

Comment:
```suggestion
        CreateNamingScreen(ctx->taskman, 5, 0, OT_NAME_LENGTH, 0, r5, retPtr);
```
As for the `5`, we should probably turn that into a `NAMINGSCREEN_UNK5` constant for now

Reply:
Maybe `NAMINGSCREEN_GROUP`?